samples/snippets/src/main/java/com/example/bigquerystorage/WriteToDefaultStream.java

```diff
@@ -22,6 +22,8 @@
 import com.google.api.core.ApiFutures;
 import com.google.api.gax.core.FixedExecutorProvider;
 import com.google.api.gax.retrying.RetrySettings;
+import com.google.api.gax.rpc.FixedHeaderProvider;
```

Comment:
Setting useragent is not a common practice. Let's keep the e2e test instead.

Reply:
I have moved the sample code that demonstrates how to set the UserAgent to an existing e2e test, namely ITBigQueryWriteManualClientTest.java.